### DISCUSSION TITLE
How can I apply CSS "hook" classes in filament

### DISCUSSION THREAD

**Discussion by alkoumi** [filamentphp/filament#8910] [Help]
I want to apply some CSS to the `fi-sidebar-group-label` and `fi-sidebar-nav` hook classes
as mentioned here [Applying styles to hook classes](https://filamentphp.com/docs/3.x/support/style-customization)

I read the Documentation But can't handle this 
How can Change CSS styles with these hooks 
If there are some examples

I hope anyone can Help!
Thanks in advance

**Comment by sheeehy** [comment]
If you want to simply target hooks like fi-sidebar-group-label and fi-sidebar-nav, you can do so by adding custom CSS in your stylesheets. You can place this CSS code in your main stylesheet, or you can include it using a <style> tag in your Blade template where your Filament UI is rendered.

If you've customized Filament by publishing its assets and have access to its Sass files, you can add your custom styles there instead. Just make sure to compile the assets afterward.

**Reply by leandrocfe** [reply]
Did you [create a custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme)?

**Reply by alkoumi** [reply]
Thank You Brothers @sheeehy and  @leandrocfe for Helping

I didn't use a custom theme and didn't publish the views 
I am following the best practices as mentioned in the Documentation here   [Applying styles to hook classes](https://filamentphp.com/docs/3.x/support/style-customization)

But I didn't find any code example for applying some CSS hook classes
same as the part of applying render HTML hooks 
```
FilamentView::registerRenderHook(
    'panels::body.start',
    fn (): View => view('impersonation-banner'),
);
```

I want to know the way or code example for applying some CSS hook
